client/me/purchases/remove-purchase/index.jsx

```diff
@@ -262,8 +269,11 @@ class RemovePurchase extends Component {
 						} )
 					}{ ' ' }
 					{ translate(
-						'You will not be able to reuse it again without purchasing a new subscription.',
-						{ comment: "'it' refers to a product purchased by a user" }
+						'You will not be able to reuse %(productName)s again without starting a new %(domainProductName)s subscription.',
```

Comment:
This code runs for more than just domain products, so `domainProductName` is probably not the best variable name.  But also, the wording doesn't seem to work well for those.  Here's what I get when removing a plan with this pull request:

![remove-plan](https://user-images.githubusercontent.com/235183/129275037-aaba3e0a-1f57-439a-88fd-83da0663c02f.png)

It's a big repetitive :smile: 

I still wonder if we might be better borrowing text from the remove+refund case instead.  I'll comment on the issue with more details about that.

Reply:
Yeah, I don't like that either... I _thought_ this was catching just the domain mapping, but I see that I need to relook at this.